src/wallet/walletdb.cpp

```diff
@@ -756,7 +757,7 @@ void MaybeCompactWalletDB()
         return;
     }
 
-    for (CWallet* pwallet : GetWallets()) {
+    for (CWallet* pwallet : g_wallet_manager.GetWallets()) {
```

Comment:
BTW, I'm planning to move the iteration to `WalletManager` and make `MaybeCompactWalletDB(WalletDatabase& dbh)`. This way `WalletManager` dependency is removed here.